.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,112 @@
+name: Package Bug Report
+description: Something went wrong with the installation of a package? Or did you find an unexpected behavior?
+title: "(packageName) "
+labels: ["01 - _Triaging", "Bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for taking the time to fill out this bug report! Please fill out out the necessary information as complete as you can. Remember, it is better to include too much information, than too little.
+        Additionall, make sure that the issue title starts with the name of the package you was seeing the issue with inside paranthesis.
```

Comment:
```suggestion
        Thank you for taking the time to fill out this bug report! Please fill out out the necessary information as complete as you can. Remember, it is better to include too much information, than too little. Additionally, make sure that the issue title starts with the name of the package you were seeing the issue with inside paranthesis (e.g. `(putty)`).
```

Reply:
I noticed that the formatting when [testing the issue](https://github.com/AdmiringWorm/chocolatey-coreteampackages/issues/new?assignees=&labels=01+-+_Triaging%2CBug&template=bug-report.yml&title=%28packageName%29+) was broken as the 'Additionally' does start on a new line but there is no spacing between the lines so it looks off. So I put them onto the same line here. You could use that or put a space between the lines.